shared/js/background/classes/tab.es6.js

```diff
@@ -44,6 +44,7 @@ class Tab {
         this.requestId = tabData.requestId
         this.status = tabData.status
         this.site = new Site(this.url)
+        this.httpsUpgrades = {}
```

Comment:
I ended up storing this here on the tab object, although I probably could have also done it on the site object. @andrey-p curious where you think the best place to store a short-lived cache like this is.

Reply:
Yeah this is probably the place for it.